msg/ServoReport.msg

```diff
@@ -0,0 +1,9 @@
+uint64 timestamp					# time since system start (microseconds)
+
+float32 servo_position					# feedback position of servo. [-1;1]
+float32 servo_speed					# feedback speed of servo [deg/s]
```

Comment:
Is it not enough to derive the speed from the position?

Reply:
yes true, but  since the servo provides feedback about its speed, we  should at least log it.